queue_services/entity-filer/src/entity_filer/worker.py

```diff
@@ -78,6 +78,7 @@
 
 
 qsm = QueueServiceManager()  # pylint: disable=invalid-name
+flags = Flags()  # pylint: disable=invalid-name
```

Comment:
The FF setup is duplicate with that I merged last week(line 88). Can you remove it?

Reply:
Removed